support/ext/event

```diff
@@ -6,6 +6,10 @@ php_version=${1:=7.4}
 
 event_version="3.0.5"
 
+if [[ ! -d /app/vendor/libwebp ]]; then
+  /buildpack/support/get_webp
+fi
```

Comment:
This is mandatory for scalingo-20

Reply:
Why is it mandatory for scalingo-20 ?

Reply:
Compilation fails without it. Error message:

```
Parse /tmp/phpevent_DHjn/event-3.0.5/php8/php_event.stub.php to generate /tmp/phpevent_DHjn/event-3.0.5/php8/php_event_arginfo.h
/app/vendor/php/bin/php: error while loading shared libraries: libwebp.so.7: cannot open shared object file: No such file or directory
make: *** [Makefile:190: /tmp/phpevent_DHjn/event-3.0.5/php8/php_event_arginfo.h] Error 127              
```

Reply:
Hmm okay. Thanks 👍🏻

Reply:
Maybe specify that requirement in the script